package-lock.json

```diff
@@ -1,5828 +1,8 @@
 {
   "name": "kappa-com",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
```

Comment:
we added only stripe as dependency then why this file contains so many changes only stripe change makes sense, please check why this contains so many changes.

Reply:
this is because of different node versions between mine and alex's systems. He was the last person who added a package.

Reply:
oh ok so can we do  rm -rf node_modules and npm cache clean and then do fresh install.